README.md

```diff
@@ -290,6 +290,26 @@ I18nModule.forRootAsync({
 });
 ```
 
+#### Using Fallbacks
+
+To configure multiple fallbacks use `fallbacks` option. You could handle a single language or multiple ones.
+
+```typescript
+I18nModule.forRoot({
+  fallbackLanguage: 'en',
+  fallbacks: {
+    'en-CA': 'fr',
+    'en-*': 'en',
+    'fr-*': 'fr',
+    'pt': 'pt-BR',
+  },
+  parser: I18nJsonParser,
+  parserOptions: {
+    path: path.join(__dirname, '/i18n/'),
+  },
+});
+```
+
```

Comment:
Sample looks good 👍 😄 , but I would add something to tell the developer that `en` and `fr` translations are needed in order for this to work correctly.

Reply:
You are right mate! 😃  I've done a small change to add this one so I can change if you want.

Reply:
Thanks mate!